fieldservice_timeline/__init__.py

```diff
@@ -0,0 +1,4 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
@AmmarOfficewalaSerpentCS Correct the copyrights in all files, please?

# Copyright (C) 2021 Open Source Integrators
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).